### DISCUSSION TITLE
Welcome to Unity 2D RPG Series 2022 Discussions!

### DISCUSSION THREAD

**Discussion by raptr45** [oneManDev-io/Make-an-RPG-Game-in-Unity-2022#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by archeryfox** [comment]
Wow! Thank your series, i have understand unity a little bit one nigth before exam
